lib/MusicBrainz/Server/Data/Recording.pm

```diff
@@ -154,6 +154,18 @@ sub find_by_release
     $self->query_to_list_limited($query, [$release_id], $limit, $offset);
 }
 
+sub find_by_work
+{
+    my ($self, $work_id, $limit, $offset) = @_;
+    my $query = "SELECT " . $self->_columns . "
+                 FROM ". $self->_table . "
+                   JOIN l_recording_work lrw ON lrw.entity0 = recording.id
```

Comment:
[Coding style](https://musicbrainz.org/doc/Development/Server_Coding_Style): should be 4 spaces for indentation rather than 2.